requirements.in

```diff
@@ -0,0 +1,7 @@
+-c constraints.in # <-- contains known limitations
+
+staticjinja==4.1.3
+Jinja2==3.1.2
+PyYAML==6.0
+libsass==0.22.0
+pathlib
```

Comment:
Why is this deprecated backport of the stdlib module needed? Would it shadow the more up-to-date stdlib version? It should probably be exterminated here and now.

Reply:
@webknjaz I don't have a good reason for why this is in here tbh. I guess obviously there is no good reason so I'll just get rid of it. thanks a mill once again. even if you're spotting some glaring errors here I'm finding the review super helpful.